docs/src/pages/components/data-grid/columns/BasicColumnsGrid.tsx

```diff
@@ -1,18 +1,18 @@
 import * as React from 'react';
-import { DataGrid, GridColDef } from '@material-ui/data-grid';
+import { DataGrid } from '@material-ui/data-grid';
 
 export default function BasicColumnsGrid() {
   return (
     <div style={{ height: 250, width: '100%' }}>
       <DataGrid
         columns={
-          [{ field: 'id' }, { field: 'username' }, { field: 'age' }] as GridColDef[]
```

Comment:
the way it's done here is not great. If we want to show how to use the type then it is better to declare a variable outside the scope with the correct type. Otherwise I think it's better without the type